src/modules/coreHttpApi/openapi.yml

```diff
@@ -3709,7 +3542,7 @@ paths:
                   type: string
                   format: date-time
                 ephemeral:
-                  description: If set to true the token will will not be cached in the database of the Connector. Note that you will not be able to fetch this token unless you remember the truncatedReference of the token. Defaults to true. Will be ignored if the accept header is set to `image/png`.
+                  description: If set to true the token will will not be cached in the database of the Connector. Note that you will not be able to fetch this token unless you remember the reference.truncated of the token. Defaults to true. Will be ignored if the accept header is set to `image/png`.
```

Comment:
Please change `reference.truncated` to `reference` everywhere in this file. We don't care if the url or truncated value is saved / used!